bin/cheribsdtest/cheribsdtest.c

```diff
@@ -734,6 +735,45 @@ cheribsdtest_spawn_child(enum spawn_child_mode mode)
 	return (pid);
 }
 
+static const char *
+_cheribsdtest_get_helper_path(const struct cheri_test *ctp)
+{
+	static char helper_path[MAXPATHLEN+1];
```

Comment:
```suggestion
	static char helper_path[MAXPATHLEN];
```

`PATH_MAX` is POSIX, but `MAXPATHLEN` is just a duplicate, and both already include the nul terminator.

Reply:
Fixed the case I'd copied from too